src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        return new Ball(colorSupplier.getRandomColor(),random.nextInt(100));
```

Comment:
Magic numbers should be avoided as per the checklist. The number `100` should be extracted into a constant with a meaningful name. For example, `private static final int MAX_NUMBER = 100;`, and then you can use `random.nextInt(MAX_NUMBER)`.